pom.xml

```diff
@@ -274,7 +274,11 @@
                     </argLine>
                 </configuration>
             </plugin>
-
+            <plugin>
+                <groupId>io.spring.javaformat</groupId>
+                <artifactId>spring-javaformat-maven-plugin</artifactId>
+                <version>0.0.38</version>
+            </plugin>
```

Comment:
We aren't adopting this plugin. Essentially, I am using https://github.com/spring-projects/spring-data-build/tree/main/etc/ide as the basis for any/all polishing going forward.

Reply:
No offense,  I just accidentally committed the pom ...